config/__init__.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+'''This module handles application configuration and secrets.'''
+
+import os
+from datetime import datetime
+from collections import namedtuple
```

Comment:
Updated, thanks (btw which linter are you using to show you this? pep8 doesn't seem to care--would be nice to catch this stuff before committing it)